docs/Course Wiki/ECE Course Offerings/ECE120.md

```diff
@@ -0,0 +1,55 @@
+# ECE120
+
+## Content Covered
+
+- Representation of information
+- Boolean algebra
+- Combinational network analysis and design
+- Sequential network analysis and design
+- Finite state machine and memory design
+- Computer organization and machine-level programming
+- Programming in C
+
+The course starts by covering the many different representations of information in the form of bits in a computing system, then moves on to cover Boolean algebra as it relates to digital networks, combinational and sequential circuits, finite state machines and memories, and ends with a basic overview of computer organization and assembly programming principles using a basic educational ISA known as LC-3. Some C programming is also covered in the first few weeks through a series of small labs, but no prior programming experience is needed and mastery of C is not expected by the end of this course, as [ECE220](ECE220.md) (the next course in line after ECE120) starts from where this course leaves off and covers C in much more detail. 
+
+## Prerequisites
+
+There are no technical prerequisites to take ECE120 as it is an introductory course to computing fundamentals, and aims to provide an overview of all the content encapsulated by the field, which future classes will expand on by diving deeper into specific topics, such as low level programming (ECE220), [ECE391](ECE391.md)), digital design ([ECE385](ECE385.md)), and computer organization ([ECE411](ECE411.md)).
+
+## When to Take it
+
+ECE120 is generally taken by computer engineering students in their first semester of freshman year, and by electrical engineering students in their second semester of freshman year. It is recommended to take ECE120 as soon as possible, especially if the student is in computer engineering, in order to open up future coursework in the computer engineering curriculum, as every future computer engineering course builds on at least one topic covered in the class, and thus it is a prerequisite to taking all of them, starting with ECE220 right after this course.
+
+NOTE: While electrical engineering students will be better suited to take ECE120 in the spring to make room early on for [ECE110](ECE110.md) and their other classes, it should be taken into consideration that [ECE210](ECE210.md) (the follow-up class for ECE110) has more prerequisites than ECE220. Thus, an electrical engineer may find it a better option to take ECE120 their first semester anyways, just so they can double up on ECE110 and ECE220 while they complete the prerequisites for ECE210. Of course, ECE210 is a fundamental EE course and should be taken as soon as possible, so plan accordingly. For a department-provided suggested schedule, refer to the [EE Curriculum](../../Curriculum%20Graph/1.EE%20Curriculum.md) section.
```

Comment:
Can this be more concise?

Reply:
I have made it a bit more concise on the latest commit, but if needed I can try and cut down more.